### PR TITLE
Added Voron SW SB CR Touch (No Tools) Mod

### PR DESCRIPTION
**Pull request by prasadp4009**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
CR Touch Mod for Voron SwitchWire with Stealtburner (Modded the rail mount with press fit CR Touch)
#### How was it tested? 
Built Voron Swithcwire (EnderWire) with the Mod
#### Any background context you want to provide?
Wanted to reuse CR Touch in place of probe in SB rail mount
#### Screenshots (if appropriate)
https://www.reddit.com/r/voroncorexy/comments/whicbz/serial_request_voron_switchwire_enderwire/?utm_source=share&utm_medium=web2x&context=3

### CODE REVIEW

File: printer_mods/prasadp4009/SW_SB_CRTouch/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Voron SwitchWire StealthBurner CR Touch Mod
```
Comment (MapleLeafMakers) [inline printer_mods/prasadp4009/SW_SB_CRTouch/README.md:1]: Switchwire should be written without a capital W.  just `Switchwire`

File: printer_mods/prasadp4009/SW_SB_CRTouch/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+This repo contains the STL for Voron SW SB CR Touch Mod.
+
+I used the SB 1.05Beta during development. But it should work on the SB v1 release as its only X rail carraige.
```
Comment (MapleLeafMakers) [inline printer_mods/prasadp4009/SW_SB_CRTouch/README.md:5]: Carriage is misspelled here.

File: printer_mods/prasadp4009/SW_SB_CRTouch/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Voron SwitchWire StealthBurner CR Touch Mod
+
+This repo contains the STL for Voron SW SB CR Touch Mod.
+
+I used the SB 1.05Beta during development. But it should work on the SB v1 release as its only X rail carraige. 
+
+- The STL is only for MGN12 Carraige for SW
```
Comment (MapleLeafMakers) [inline printer_mods/prasadp4009/SW_SB_CRTouch/README.md:7]: Carriage is also misspelled here

File: printer_mods/prasadp4009/SW_SB_CRTouch/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+- No supports are needed during the print
+- No tools are needed to fit the CR Touch, its a **press fit**
```
Comment (MapleLeafMakers) [inline printer_mods/prasadp4009/SW_SB_CRTouch/README.md:9]: `it's` not `its`

File: printer_mods/prasadp4009/SW_SB_CRTouch/README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+- I used Dragon Standard hotend for deciding height of CR touch and its 2mm above the nozzle when pin is up
```
Comment (MapleLeafMakers) [inline printer_mods/prasadp4009/SW_SB_CRTouch/README.md:10]: `it's`

**Review by MapleLeafMakers — COMMENTED** [review]
Can you remove the LICENSE file please? all mods in the repo are licensed under the GPL3

File: printer_mods/prasadp4009/SW_SB_CRTouch/README.md (hunk 1 above)
Comment (prasadp4009) [reply printer_mods/prasadp4009/SW_SB_CRTouch/README.md]: Fixed it.

File: printer_mods/prasadp4009/SW_SB_CRTouch/README.md (hunk 2 above)
Comment (prasadp4009) [reply printer_mods/prasadp4009/SW_SB_CRTouch/README.md]: Fixed it.

File: printer_mods/prasadp4009/SW_SB_CRTouch/README.md (hunk 3 above)
Comment (prasadp4009) [reply printer_mods/prasadp4009/SW_SB_CRTouch/README.md]: Fixed it.

File: printer_mods/prasadp4009/SW_SB_CRTouch/README.md (hunk 4 above)
Comment (prasadp4009) [reply printer_mods/prasadp4009/SW_SB_CRTouch/README.md]: Fixed it.

File: printer_mods/prasadp4009/SW_SB_CRTouch/README.md (hunk 5 above)
Comment (prasadp4009) [reply printer_mods/prasadp4009/SW_SB_CRTouch/README.md]: Fixed it.

**Review by cbc02009 — APPROVED** [review]

**Comment by cbc02009** [comment]
Looks like this has some merge conflicts that need fixing before it can be added to the repo. Let us know if you're not sure how to fix it and we can walk you through it.

**Comment by prasadp4009** [comment]
Is there anything I need to do on my end?

**Comment by cbc02009** [comment]
Yes, there are merge conflicts in the main readme file that need fixing.

**Comment by prasadp4009** [comment]
Fixed the README.md

**Comment by prasadp4009** [comment]
Is there anything left?

**Comment by prasadp4009** [comment]
Fixed the README.md
Request to merge.

**Comment by prasadp4009** [comment]
Is there anymore change needed?

**Comment by prasadp4009** [comment]
> Sorry, I screwed up the order I merged the mods in, and now the printer/readme.md needs to be fixed again. Once that's fixed I can merge it in.

Done. You should be able to merge it.

**Review by prasadp4009 — COMMENTED** [review]
All changes complete.

**Comment by prasadp4009** [comment]
@cbc02009 Can you please review? Thanks!

**Pull request merged**